swagger.yaml

```diff
@@ -28,40 +28,106 @@ securityDefinitions:
       routes.write_regex: |
         All operations on all routes, including ones with regex path matching.
 paths:
-  '/routes':
-    x-swagger-router-controller: "routes"
+  '/hosts':
     get:
-      operationId: getRoutes
       security:
         - company:
           - routes.read
-      summary: Returns the list of all the routes.
+      summary: Returns the list of available hosts.
+      responses:
+        '200':
+          description: An array of available hosts.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Host'
+        default:
+          description: A processing or an unexpected error.
+          schema:
+            $ref: '#/definitions/Error'
+  '/paths':
+    post:
+      security:
+        - company:
+          - routes.write_full_path
+      summary: Creates a new path resource that belongs to the team of the creator.
       parameters:
-        - name: name
+        - name: path
+          in: body
+          description: Path to create.
+          required: true
+          schema:
+            $ref: '#/definitions/PathIn'
+      responses:
+        '201':
+          description: The path is created.
+          schema:
+            $ref: '#/definitions/PathOut'
+        default:
+          description: A processing or an unexpected error.
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      security:
+        - company:
+          - routes.read
+      summary: A list of available paths.
+      parameters:
+        - name: team
           in: query
-          description: The name of the route to filter for
+          type: string
           required: false
+        - name: uri
```

Comment:
This is not the final version. Let's add it later, when we will have requests for this feature.